merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/exceptions/NoSuchActivityInstanceException.java

```diff
@@ -1,16 +1,18 @@
 package gov.nasa.jpl.aerie.merlin.server.exceptions;
 
+import gov.nasa.jpl.aerie.merlin.driver.ActivityInstanceId;
+
 public class NoSuchActivityInstanceException extends Exception {
   private final String planId;
-  private final String activityInstanceId;
+  private final ActivityInstanceId activityInstanceId;
 
-  public NoSuchActivityInstanceException(final String planId, final String activityInstanceId) {
+  public NoSuchActivityInstanceException(final String planId, final ActivityInstanceId activityInstanceId) {
     super("No activity exists with id `" + planId + "` in plan with id `" + planId + "`");
```

Comment:
The first use of `planId` should be `activityInstanceId`, actually -- amusing that we never saw this 😆

Reply:
(I think `ActivityInstanceId` is a `record`, but if it isn't, please make sure it has a `toString method`.)

Reply:
I'll fix that in my second PR where I'm adding a `PlanId` record!